perllib/FixMyStreet/Cobrand/Bromley.pm

```diff
@@ -768,6 +768,28 @@ sub bin_services_for_address {
     return \@out;
 }
 
+sub _get_current_service_task {
+    my $service = shift;
+
+    my $task = $service->{ServiceTasks}{ServiceTask};
+    my $type = ref $task;
+    return $task if $type eq 'HASH';
+
+    my ($current, $last_date);
+    if ( $type eq 'ARRAY' ) {
+        foreach ( @$task ) {
+            my $end = construct_bin_date($_->{EndDate});
+
+            next if $end && $end < $last_date;
```

Comment:
should this be checking $last_date as well, or it'll have an undefined warning on first pass of loop.